content/administration/on_premise/deploy.rst

```diff
@@ -600,6 +600,145 @@ It should be stored securely, and should be generated randomly e.g.
 
 which will generate a 32 characters pseudorandom printable string.
 
+Reset the master password
+-------------------------
+
+There may be instances where the master password is misplaced or compromised and needs to be reset.
+The following process is for system administrators of an Odoo on-premise database to manually reset
+and re-encrypt the master password.
+
+.. seealso::
+   For more information about changing an Odoo.com account password see this documentation:
+   :ref:`odoocom/change_password`.
+
+When creating a new on-premise database, a random master password is generated. Odoo recommends
+using this password to secure the database. Odoo implements this password by default so that there
+is a secure master password for any Odoo on-premise deployment.
+
+.. note::
+   When creating an Odoo on-premise database the installation is accessible to anyone on the
+   internet until this password is set to secure the database.
+
+The master password is specified in the Odoo configuration file (`odoo.conf` or `odoorc` (hidden
+file)). The Odoo master password is needed to modify, create, or delete a database through the
+graphical user interface (GUI).
+
+Locate configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, :guilabel:`Open` the Odoo configuration file (`odoo.conf` or `odoorc` (hidden file)).
+
+.. tabs::
+
+   .. tab:: Windows
+
+      The configuration file is located at: `c:\\ProgramFiles\\Odoo{VERSION}\\server\\odoo.conf`
+
+   .. tab:: Linux
+
+      Depending on how Odoo is installed on the Linux machine the configuration file is located in
+      different places:
+
+      - Package installation: `/etc/odoo.conf`
+      - Source installation: `~/.odoorc`
+
+Change old password
+~~~~~~~~~~~~~~~~~~~
+
+Then, modify the old password in the configuration file to a temporary password.
+
+.. tabs::
+
+   .. tab:: Graphical user interface
+
+      After locating the configuration file open it using a graphical user interface (GUI). This can
```

Comment:
```suggestion
      After locating the configuration file, open it using a graphical user interface (GUI). This can
```
Since we've established what GUI means before this, an :abbr: tag could work here instead. Optional obviously depending on what you want